src/site/content/en/handbook/grammar/index.md

```diff
@@ -259,7 +265,7 @@ Add a space before units.
 ## Usage
 Use consistent vocabulary throughout a piece and the collection it lives in.
```

Comment:
I don't know what "consistent vocab" means here. Like, if you use a term, use that same term throughout? Or consistently use acronyms?

Reply:
Yup, that's what it means :)  This may seem obvious, but it's been a problem.

Suggestions for better phrasing?

Reply:
Rephrased in the Google Doc and added an example.